buildbot/osuosl/master/config/builders.py

```diff
@@ -442,6 +442,7 @@
     'tags'  : ["clang"],
     'workernames' : ["linaro-g3-01", "linaro-g3-02", "linaro-g3-03", "linaro-g3-04"],
     'builddir': "clang-aarch64-sve-vla",
+    'max_simultaneous_builds' : 1,
```

Comment:
I wonder if there are any documentation comments somewhere that we can add an explanation of this to.

Reply:
Mostly to mention that the "default" is unlimited jobs per builder.